ruma-federation-api/src/keys/claim_keys/v1.rs

```diff
@@ -0,0 +1,38 @@
+//! [POST /_matrix/federation/v1/user/keys/claim](https://matrix.org/docs/spec/server_server/r0.1.4#id91)
```

Comment:
Ah, so copying the link of a heading doesn't actually give you an anchor to the heading itself but to that heading in the table of contents. It really annoying for this kind of thing, but you'll notice that all actual endpoint definition anchors have the endpoint name as the anchor rather than just `id{N}`.